decode/render.ts

```diff
@@ -3,10 +3,12 @@ import { DomData } from "../types/decode/layout";
 import { InputChangeData, PointerData, ResizeData, ScrollData, SelectionData } from "../types/interaction";
 import { BoxModelData } from "../types/layout";
 
+const SVG_NAMESPACE: string = "http://www.w3.org/2000/svg";
```

Comment:
Don't deduplciate this here. Define it one place and reference. It's in at least 3 files of the code. 
I'm very strong about "DRY" (don't repeat yourself). Hence harping on this.

Reply:
Done. Refactored into const enum.